movie/ddl.go

```diff
@@ -43,12 +43,12 @@ func (w *ddlManager) createTables(ctx context.Context) error {
 	// Movie.
 	query := `
 		CREATE TABLE IF NOT EXISTS movie (
-			id BIGINT NOT NULL,
-			title VARCHAR(128),
-			type VARCHAR(20),
-			year SMALLINT,
-			released_at DATETIME,
-			PRIMARY KEY (id)
+			id bigint(20) NOT NULL,
+			title varchar(100) NOT NULL,
+			type enum('Action','Adventure','Animation','Children''s','Comedy','Crime','Documentary','Drama','Fantasy','Film-Noir','Horror','Musical','Mystery','Romance','Sci-Fi','Thriller','War') NOT NULL,
+			year smallint(6) NOT NULL,
+			release_time datetime NOT NULL,
```

Comment:
Considering that the rating time is suffixed with `at`(`rating_at`), I want to unify this naming style.

Reply:
OK